client/core/notification.go

```diff
@@ -556,7 +556,7 @@ const (
 
 func newServerNotifyNote(topic Topic, subject, details string, severity db.Severity) *ServerNotifyNote {
 	return &ServerNotifyNote{
-		Notification: db.NewNotification(NoteTypeServerNotify, topic, subject, details, severity),
+		Notification: db.NewNotification(NoteTypeNotify, topic, subject, details, severity),
 	}
 }
```

Comment:
I was thinking about having a single `NotifyNote` for this and the new `newClientNotifyNote` below . Any thoughts?

Reply:
No.  All these notes are originating from the client, and only the _one_ that is forwarded from the server requires any qualification on the source.
`ClientNotifyNote` doesn't make sense to me.  This PR is adding a notification regarding connectivity with a server.  That is the kind of note to define (e.g. `DexConnectivityNote`), if it has to be a new type, which is questionable.  I suspect there's already a fairly generic  note type that can be given an appropriate `subject` for this case.

Reply:
Okay, there's `ConnEventNote`  which I think will be suitable. I had intended not to bind connectivity messages to a specific dex connection.